package/MDAnalysis/coordinates/base.py

```diff
@@ -2078,6 +2078,20 @@ def __init__(self, filename, convert_units=None, **kwargs):
 
         self._ts_kwargs = ts_kwargs
 
+    @classmethod
+    def _unpickle_Reader(cls, filename, timestep, auxs, trans):
```

Comment:
You are not using `timestep` here. Is this "for future use"? Then please document.

Reply:
Ah no, I just forgot to use it.  There's potentially some state in `ts`, (ie if someone edits positions this is temporarily stored there)

Reply:
Reader just seems to work magically....